vital/CMakeLists.txt

```diff
@@ -173,19 +175,20 @@ set( vital_sources
   io/track_set_io.cxx
 
   types/activity.cxx
+  types/activity_type.cxx
   types/bounding_box.cxx
   types/camera_intrinsics.cxx
   types/camera_perspective.cxx
   types/camera_rpc.cxx
   types/category_hierarchy.cxx
-  types/class_map.cxx
```

Comment:
Should we be adding types/class_map.txx here?

Reply:
No; a `.txx` can't be compiled on its own (or it would be a `.cxx`). A `.txx` is a specific type of (semi-private) header which provides template definitions — separate from a header which provides only declarations — for templates that are intended to be used solely or primarily via explicit instantiation. In such cases, the `.txx` is included (as in this PR) by those `.cxx` files (e.g. [`activity_type.cxx`](https://github.com/Kitware/kwiver/blob/8d894e9bad0f3dd366b56d7702aa3d40fe179553/vital/types/activity_type.cxx#L33), [`detected_object_type.cxx`](https://github.com/Kitware/kwiver/blob/8d894e9bad0f3dd366b56d7702aa3d40fe179553/vital/types/detected_object_type.cxx#L33)) which provide explicit instantiations.